src/screen/GameScreen.java

```diff
@@ -179,14 +179,14 @@ public final void initialize() {
 		enemyShipFormation = new EnemyShipFormation(this.gameSettings, this.gameState);
 		enemyShipFormation.attach(this);
         // Appears each 10-30 seconds.
-        this.ship = ShipFactory.create(this.shipType, this.width / 2, this.height - 30);
+        this.ship = ShipFactory.create(this.shipType, this.width / 2, this.height - 70);
         ship.applyItem(wallet);
 		//Create random Spider Web.
 		int web_count = 1 + level / 3;
 		web = new ArrayList<>();
 		for(int i = 0; i < web_count; i++) {
 			double randomValue = Math.random();
-			this.web.add(new Web((int) Math.max(0, randomValue * width - 12 * 2), this.height - 30));
+			this.web.add(new Web((int) Math.max(0, randomValue * width - 12 * 2), this.height - 70));
```

Comment:
Is there a reason why the Y coordinates were changed? I'm curious if you have any!

Reply:
I faced an issue where the ship would simply be out of the screen, and only by increasing the offset would I be able to see it properly. I also increased it for the webs accordingly.

Reply:
> I faced an issue where the ship would simply be out of the screen, and only by increasing the offset would I be able to see it properly. I also increased it for the webs accordingly.

Can you tell me exactly under what circumstances such an out-screen bug occurs?

Reply:
This is the preview before adding the offset:
[Screencast from 2024-10-12 21-59-51.webm](https://github.com/user-attachments/assets/57a221d3-b504-4a04-ac0e-2d6183afb457)

And now after adding it:
[Screencast from 2024-10-12 22-00-22.webm](https://github.com/user-attachments/assets/a99cee8b-45ed-4c2d-a31c-05e8d1f177af)